keps/sig-network/1880-multiple-service-cidrs/README.md

```diff
@@ -697,7 +682,7 @@ it will be safe to disable the dual-write mode.
 | 1.31     | Beta off | Alpha off   |
 | 1.32     | Beta on  | Alpha off   |
 | 1.33     | GA on    | Beta off   |
-| 1.34     | GA (there are no bitmaps running) | GA on (also delete old bitmap)|
+| 1.34     | GA (there are no bitmaps running) | GA (also delete old bitmap)|
 | 1.35     | remove feature gate | remove feature gate  |
```

Comment:
With the new changes coming from [Compatibility Versions](https://github.com/kubernetes/enhancements/issues/4330) this should be bumped to 1.36 for `MultiCIDRServiceAllocator` and 1.37 for `DisableAllocatorDualWrite` since each has to be tweakable for 3 releases after GA.